tests/baselines/reference/defaultExportsCannotMerge04.js

```diff
@@ -18,7 +18,7 @@ export interface Foo {
 function Foo() {
 }
 Object.defineProperty(exports, "__esModule", { value: true });
-exports.default = Foo;
+exports.default = exports.Foo;
 var Foo;
 (function (Foo) {
-})(Foo || (Foo = {}));
+})(exports.Foo || (exports.Foo = {}));

```

Comment:
the issue is we do not expect something to be exported with its name and as default. it is either this or the other.